config/compareWarpAssembleCoadd.py

```diff
@@ -0,0 +1,15 @@
+import os.path
+
+# Load configs from base assembleCoadd
+config.load(os.path.join(os.path.dirname(__file__), "assembleCoadd.py"))
+
+# 200 rows (since patch width is typically < 10k pixels
+config.assembleStaticSkyModel.subregionSize = (10000, 200)
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)